index.html

```diff
@@ -0,0 +1,131 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="media-style.css">
+    <title>Anicet's Capstone Project</title>
+</head>
+
+<body>
+    <header>
+        <div class="logo"><a href=""><img src="./images/NFTSbck.jpg" alt="This a NFT logo"></a></div>
+
+        <ul>
+            <li><a href="About.html">About</a></li>
+            <li><a href="">Progran</a></li>
+            <li><a href="">Join</a></li>
+            <li><a href="">Sponsors</a></li>
+            <li><a href="">News</a></li>
+            <li><a href="">CC campaign</a></li>
+        </ul>
+    </header>
```

Comment:
The navigation is meant to be divided in a top dark bar and a bottom white bar.

- [ ] The top short bar section with social media links, logout, and language links aligned to the right which is currently missing in your project. Kindly add it to your project as seen in the original design below 👇🏻 

![image](https://user-images.githubusercontent.com/4247740/158913253-7f9d20d5-5ebc-4e84-bc2f-d578616d010d.png)

- [ ] I also suggest the use of more semantic tags by applying the `<nav>`. For example you could use the structure below
```
    <header>
        <a href="">
            <img src="./images/NFTSbck.jpg" alt="This a NFT logo">
        </a>
       <nav>
            <ul>
                <li><a href="About.html">About</a></li>
                ...
            </ul>
        </nav>
    </header>
```

- [ ] I have also noticed the `CC Campaign` link is not properly styled as seen in the original design. Please fix it style-wise

![image](https://user-images.githubusercontent.com/4247740/158913960-63eef3c0-7c0c-410b-9dc5-3773726f6286.png)